tests/testthat/test-utils.R

```diff
@@ -142,3 +142,16 @@ test_that("try and find a free port", {
     fixed = TRUE)
   mockery::expect_called(mock_check_port, 5)
 })
+
+
+test_that("can format difftime nicely", {
+  now <- Sys.time()
```

Comment:
consider making the test deterministic by doing:

```suggestion
  now <- ISOdate(2022, 11, 11, 10, 49, 33.123)
```